### PR TITLE
[LOOP-Hotfix] COB should always be check and displayed

### PR DESCRIPTION
**Pull request by nhamming**
always check for cob to display

### CODE REVIEW

**Review by darinkrauss — APPROVED** [review]
LGTM! 🏎️

File: Loop/View Controllers/StatusTableViewController.swift [hunk 1]
```diff
@@ -387,9 +387,10 @@ final class StatusTableViewController: LoopChartsTableViewController {
                     }
                     reloadGroup.leave()
                 }
-                carbsOnBoard = state.carbsOnBoard?.quantity
             }
-
+            // always check for cob
+            carbsOnBoard = state.carbsOnBoard?.quantity
```
Comment (ps2) [inline Loop/View Controllers/StatusTableViewController.swift:392]: What was causing cob to update without .carbs being in the refresh context?

Comment (nhamming) [reply Loop/View Controllers/StatusTableViewController.swift:392]: It isn't updating, but that `carbsOnBoard` is set to `nil` with each reload, which causes the cob string to also be set to `nil` until carb chart points are available.

Are you suggesting that cob should be a stored property of the VC?

Comment (ps2) [reply Loop/View Controllers/StatusTableViewController.swift:392]: Ok, I see this is a new var that was added. Sounds good.

**Pull request merged**